native/package.json

```diff
@@ -6,7 +6,7 @@
     "link": "react-native link",
     "start": "react-native start",
     "ios": "react-native run-ios",
-    "android:emulator": "$ANDROID_HOME/tools/emulator -avd bpk-avd -dpi-device 420 -skin 1080x1920",
+    "android:emulator": "$ANDROID_HOME/tools/emulator -avd bpk-avd",
```

Comment:
This is change is on purpose, will prevent the Pixel2 emulator to be forced at 1080 and render the content is a weird way

Reply:
Oh awesome - I wrote this quite a while ago and it was more-or-less hacked together, so thanks for optimising it!